packages/flutter_tools/lib/src/ios/xcodeproj.dart

```diff
@@ -45,6 +52,11 @@ void updateXcodeGeneratedProperties(String projectPath, BuildMode mode, String t
     localsBuffer.writeln('LOCAL_ENGINE=${localEngineArtifacts.engineOutPath}');
   }
 
+  // If the project isn't using platform plugins, don't assume CocoaPods will be
+  // needed and depended on.
+  if (hasPlugins)
+    localsBuffer.writeln('#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.release.xcconfig"');
```

Comment:
It'll probably work, but before this we included the debug xcconfig in a debug build.